src/components/controller/appLoader.ts

```diff
@@ -1,9 +1,10 @@
 import Loader from './loader';
 
 class AppLoader extends Loader {
+   
     constructor() {
-        super(process.env.API_URL, {
-            apiKey: process.env.API_KEY,
+        super(process.env.API_URL!, {
```

Comment:
it's unsafe to do this, because someone can forget to setup env variables. It's better to check them and handle unexpected:
```typescript
const apiUrl = process.env.API_URL;
const apiKey = process.env.API_KEY;
if (!apiUrl || !apiKey) {
    throw Error("Configuration is missing")
}
super(apiUrl, apiKey);